submissions/alstep07/tiny-js-world/index.js

```diff
@@ -0,0 +1,78 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const man = {
+    species: 'human',
+    name: 'Sasha',
+    gender: 'male',
+    legs: 2,
+    hands: 2,
+    saying: 'Hello world',
+ };
+ 
+ const woman = {
+    species: 'human',
+    name: 'Julia',
+    gender: 'female',
+    legs: 2,
+    hands: 2,
+    saying: 'Hi there',
+ };
+ 
+ const cat = {
+    species: 'cat',
+    name: 'Tom',
+    gender: 'male',
+    legs: 4,
+    hands: 'no',
+    saying: 'Meow meow',
+ };
+ 
+ const dog = {
+    species: 'dog',
+    name: 'Jack',
+    gender: 'male',
+    legs: 4,
+    hands: 'no',
+    saying: 'Bow wow',
+ };
+ 
+ const catWoman = {
+    species: 'woman',
+    name: 'Selina',
+    gender: 'female',
+    legs: 2,
+    hands: 2,
+    saying: cat.saying
+ };
+ 
+ // ======== OUTPUT ========
+ 
+ const inhabitants = [man, woman, cat, dog, catWoman];
+ 
+ const message = (obj) => `${obj.saying}! I am a ${obj.species} and my name is ${obj.name}. My gender is ${obj.gender}. I have ${obj.legs} legs and ${obj.hands} hands.`
+ 
+ inhabitants.forEach( inh => print(message(inh)) ); 
```

Comment:
Please avoid non-obvious acronyms like `inh`. Take care of the code readers.

Reply:
> Please avoid non-obvious acronyms like `inh`. Take care of the code readers.

@OleksiyRudenko Thank you for review, I'll keep this in mind